HHS/Head-Start-TTADP
# 488: Add approved date and created date to approved activity report view + 492: update data anonymizer script + 416: Update Accessibility for Date Selector

## Description of change

# 488:
- Add the approved date and created date to the approved report view (the created date is hidden when printing)
- For developer experience, also locked our version of minio to a specific tagged release so that it works the same in all environments.

# 416: 
- Update aria messages on Start and End date selectors to include expected format.
- Clean up contrast (default, hover, select) on calendar selector arrow but

# 492:
Includes changes to allow removing PII for the newly added grant and program specialists. It also removes entries from the RequestErrors table

## How to test

#488:
View an approved report in staging. It should have the approved date and created date up at the top. "Print" a pdf. You should see only the approved date.

#416: 
In the application tab or click on any start or end date entry field. We should now tell the user via screen reader the field and expected format.

Open all calendar selectors in the app the month arrow selector buttons should now have a gray contrast with a hover and selection.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-488
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-416
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-492

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
